app/src/main/java/org/oppia/android/app/survey/SurveyFragment.kt

```diff
@@ -18,6 +23,9 @@ class SurveyFragment :
   SelectedAnswerHandler {
 
   companion object {
+    /** Arguments key for SurveyFragment. */
+    const val SURVEY_FRAGMENT_ARGUMENTS_KEY = "SurveyFragmentarguments"
```

Comment:
Nit: Proper camel case.
```suggestion
    const val SURVEY_FRAGMENT_ARGUMENTS_KEY = "SurveyFragmentArguments"
```